server/prisma/schema.prisma

```diff
@@ -10,12 +10,172 @@ datasource db {
   url      = env("DATABASE_URL")
```

Comment:
The migrations worked as expected. I think we just need to add to the README to include the setup for a database and a `.env` file and instructions to run the migrations etc.

Reply:
@javtran so you're not using the Docker Compose configuration to run the server? I'll open a new issue to amend the README for folks not using Docker...